### ISSUE TITLE
Dark mode support

### ISSUE THREAD

**Issue by fluxxus** [pcbowers/sanity-plugin-tags#28]
Hey, quick one. 

Unless I am missing something obvious, it looks like there is no dark mode support by default?

**Comment by liamb13** [comment]
You can pass directly to the React-Select component.

Took me a little bit to figure out, but I reversed the theme colours for it.

**Original:**
```
{
  "borderRadius": 4,
  "colors": {
    "primary": "#2684FF",
    "primary75": "#4C9AFF",
    "primary50": "#B2D4FF",
    "primary25": "#DEEBFF",
    "danger": "#DE350B",
    "dangerLight": "#FFBDAD",
    "neutral0": "hsl(0, 0%, 100%)",
    "neutral5": "hsl(0, 0%, 95%)",
    "neutral10": "hsl(0, 0%, 90%)",
    "neutral20": "hsl(0, 0%, 80%)",
    "neutral30": "hsl(0, 0%, 70%)",
    "neutral40": "hsl(0, 0%, 60%)",
    "neutral50": "hsl(0, 0%, 50%)",
    "neutral60": "hsl(0, 0%, 40%)",
    "neutral70": "hsl(0, 0%, 30%)",
    "neutral80": "hsl(0, 0%, 20%)",
    "neutral90": "hsl(0, 0%, 10%)"
  },
  "spacing": {
    "baseUnit": 4,
    "controlHeight": 38,
    "menuGutter": 8
  }
}
```

**Added the following to tags field:**
```
reactSelectOptions: {
  theme: (theme: any) => ({
    ...theme,
    colors: {
      ...theme.colors,
      "neutral0": "hsl(0, 0%, 10%)",
      "neutral5": "hsl(0, 0%, 15%)",
      "neutral10": "hsl(0, 0%, 20%)",
      "neutral20": "hsl(0, 0%, 30%)",
      "neutral30": "hsl(0, 0%, 40%)",
      "neutral40": "hsl(0, 0%, 50%)",
      "neutral50": "hsl(0, 0%, 60%)",
      "neutral60": "hsl(0, 0%, 70%)",
      "neutral70": "hsl(0, 0%, 80%)",
      "neutral80": "hsl(0, 0%, 90%)",
      "neutral90": "hsl(0, 0%, 100%)"
    },
  })
},
```

**Entire field:**
```
defineField({
  name: 'tags',
  title: 'Tags',
  type: 'tags',
  options: {
    reactSelectOptions: {
      theme: (theme: any) => ({
        ...theme,
        colors: {
          ...theme.colors,
          "neutral0": "hsl(0, 0%, 10%)",
          "neutral5": "hsl(0, 0%, 15%)",
          "neutral10": "hsl(0, 0%, 20%)",
          "neutral20": "hsl(0, 0%, 30%)",
          "neutral30": "hsl(0, 0%, 40%)",
          "neutral40": "hsl(0, 0%, 50%)",
          "neutral50": "hsl(0, 0%, 60%)",
          "neutral60": "hsl(0, 0%, 70%)",
          "neutral70": "hsl(0, 0%, 80%)",
          "neutral80": "hsl(0, 0%, 90%)",
          "neutral90": "hsl(0, 0%, 100%)"
        },
      })
    },
    includeFromRelated: 'tags'
  }
}),
```

**Comment by fluxxus** [comment]
@liamb13 Amazing, thank you! 

Sorry for late reply, guess I missed the notification or the notification missed me :)

**Comment by n-kort** [comment]
What was the fix here? The latest version does not work in dark mode, and the above suggestion does not work in either...

Default:
<img width="703" alt="Image" src="https://github.com/user-attachments/assets/ffa3eda0-63b5-4ee1-866e-da0a44dd0c6b" />

Above fix:
<img width="667" alt="Image" src="https://github.com/user-attachments/assets/d40cd361-f414-44c1-af20-0fdef73afa8e" />
<img width="662" alt="Image" src="https://github.com/user-attachments/assets/2a4416c2-671a-43ec-9e8d-42b4f540fb8d" />

**Comment by ryjac** [comment]
@n-kort 

I can't speak for the above solution proposed in the post, but I fixed the dark mode issue in #36 which went live in the current version 2.1.1.


Light Mode:

![Image](https://github.com/user-attachments/assets/1bf5ce4a-0da7-4c37-ad17-bfe852529326)

Dark Mode:

![Image](https://github.com/user-attachments/assets/88b5a2eb-d940-4800-b082-ba9dfde9f0dc)
